__PROJECT NAME__/fastlane/Developer Portal/Codefile

```diff
@@ -78,4 +78,37 @@ platform :ios do
       )
     end
   end
+
+  desc "create App ID for all app bundles"
+  lane :create_app_ids do
+    ## DEBUG
+    create_app_on_developer_portal(
+      app_identifier: "#{ENV["BUNDLE_ID_DEBUG"]}",
+      app_name: "#{ENV["SCHEME_NAME_DEBUG"]}",
+    )
+    ## STAGING
+    create_app_on_developer_portal(
+      app_identifier: "#{ENV["BUNDLE_ID_STAGING"]}",
+      app_name: "#{ENV["SCHEME_NAME_STAGING"]}",
+    )
+    ## RELEASE
+    create_app_on_developer_portal(
+      app_identifier: "#{ENV["BUNDLE_ID"]}",
+      app_name: "#{ENV["SCHEME_NAME"]}",
+    )
+  end
+
+  desc "create App ID for all app bundles."
+  private_lane :create_app_on_developer_portal do |options|
+    produce(
+      username: "#{ENV["APPLE_ID"]}",
+      app_identifier: options[:app_identifier],
+      app_name: options[:app_name].gsub("-", " "),
+      language: options[:language] || "English",
+      app_version: options[:app_version] || "1.0",
+      sku: options[:sku],
+      team_name: options[:team_name],
```

Comment:
correct if I'm wrong, but from what I understand is that all the options here will actually be ignored, right?

If so, should we add `options` as a parameter to `create_app_ids` and pass it along to this function as well? 🤔

Reply:
Yes, some options can be ignore. 

'create_app_ids' lane will be executed manually with fastlane cli. It's possible to make optional options for this lane. But it' will be hard to execute with option because we have to pass these parameters with cli and as I see parameters of produce, there's nothing we need to custom.